OnboardPietroPieri/App.tsx

```diff
@@ -5,23 +5,100 @@
  * @format
  */
 
-import React from "react";
-import {View, Text, StyleSheet} from 'react-native';
+import React, {PropsWithChildren} from "react";
+import {
+  TouchableOpacity,
+  TextInput,
+  SafeAreaView,
+  View,
+  Text,
+  StyleSheet
+} from 'react-native';
 
 const styles = StyleSheet.create({
+  safeArea: {
+    marginTop: 80,
+    flex: 1,
+  },
   center: {
+    marginTop: 30,
     alignItems: 'center',
-    flex: 1,
     justifyContent: 'center',
+    flex: 0.3,
+    paddingHorizontal: 24,
+  },
+  textBemVindo: {
+    fontSize: 30,
+    fontWeight: 'bold',
+    textAlign: 'center',
+    color: 'black',
+  },
+  inputText: {
+    color: 'black',
+    fontWeight: 'normal',
+    marginLeft: 40,
+    top: 28,
+  },
+  input: {
+    marginLeft: 40,
+    marginTop: 30,
+    height: 50,
+    borderColor: 'gray',
+    borderWidth: 2,
+    paddingHorizontal: 10,
+    width: '80%',
+    borderRadius: 10,
+  },
+  button: {
+    height: 50,
+    marginTop: 50,
+    marginLeft: 40,
+    backgroundColor: '#9403fc',
+    padding: 10,
+    borderRadius: 10,
+    width: '80%',
   },
+  buttonText: {
+    textAlign: 'center',
+    fontWeight: 'bold',
+    color: 'white',
+    fontSize: 16,
+  }
+  
+
 });
 
-const App = () => {
+type SectionProps = PropsWithChildren<{
+  title: string;
+}>;
+
+function Section({children, title}: SectionProps): React.JSX.Element {
```

Comment:
Se não for usar uma prop, não precisa tipar ela, nesse caso não usa a children, então poderia tirar

Reply:
Aplicado